docs_src/pipelinebenchmarking.md

```diff
@@ -15,43 +18,48 @@ Benchmark scripts are containerized inside docker, depending on platforms/hardwa
     Build command may take a while to run depending on your internet connection and machine specifications.
 
 ## Step 2: Run Benchmark
-The benchmark.sh shell script is located under `benchmark_scripts` directory under the base directory.  Before executing this script,
+The `benchmark.sh` shell script is located under `benchmark_scripts` directory under the base directory.  Before executing this script,
 change the current directory to directory `benchmark_scripts`.
 
+This script will start benchmarking a specific number of pipelines or can start stream density benchmarking based on the desired FPS to reach.  
+Before running pipeline benchmarking for a given use case, determine the appropriate inputs, from the list below:
+
 ### Determine the input source type
 
-### RTSP
+The benchmark script can take one of these video input sources as described below:
+
+### Real Time Streaming Protocol (RTSP)
 
     --inputsrc rtsp://127.0.0.1:8554/camera_0
 
-- **__NOTE:__** using RTSP source with the benchmark.sh will automatically run the camera simulator. The camera simulator will start an RTSP stream for each video file found in the sample-media folder.
+!!! note
+    Using RTSP source with the benchmark.sh will automatically run the camera simulator. The camera simulator will start an RTSP stream for each video file found in the `sample-media` folder.
 
 ### USB Camera
 
-    --inputsrc /dev/videoN, where N is 0 or integer number
+    --inputsrc /dev/video<N>, where N is 0 or an integer
 
 ### RealSense Camera
 
-    --inputsrc <camera serial number> 
+    --inputsrc <RealSense camera serial number>
 
 #### Obtaining RealSense camera serial number
 
-[How_to_get_serial_number](./camera_serial_number.md)
+[Follow this link to get serial number](./camera_serial_number.md)
```

Comment:
I think this would read better as "Follow this link to find how to get the serial number"